parsl/monitoring/db_local.py

```diff
@@ -0,0 +1,218 @@
+import logging
+import time
+import json
+
+try:
+    from tornado import httpclient
+    import sqlalchemy as sa
+    from sqlalchemy import Table, Column, Text
+except ImportError:
+    pass
+
+
+# TODO: expand to full set of info
+def create_workflows_table(meta):
+    return Table(
+            'workflows', meta,
+            Column('task_run_id', Text, nullable=False, primary_key=True),
+            Column('time_began', Text, nullable=False),
+            Column('time_completed', Text),
+            Column('host', Text, nullable=False),
+            Column('user', Text, nullable=False),
+            Column('rundir', Text, nullable=False),
+            Column('tasks_failed_count', Text, nullable=False),
+            Column('tasks_completed_count', Text, nullable=False),
+    )
+
+
+# TODO: expand to full set of info
+def create_task_status_table(meta):
+    return Table(
+          'task_status', meta,
+          Column('task_id', Text, sa.ForeignKey('task.task_id'), nullable=False),
+          Column('task_status', Text, nullable=False),
+          Column('task_status_name', Text, nullable=False),
+          # Column('timestamp', Text, nullable=False, primary_key=True),
+          Column('timestamp', Text, nullable=False),
+          Column('task_run_id', Text, sa.ForeignKey('workflows.task_run_id'), nullable=False),
+          Column('task_fail_count', Text, nullable=False),
+          Column('task_fail_history', Text, nullable=True),
+    )
+
+
+def create_workflow_table(meta):
```

Comment:
this creates the `task` table, right? not the `workflow` table? so rename to create_task_table?